templates/clusterclass/kcpt.yaml

```diff
@@ -40,9 +40,9 @@ spec:
                       - name: vip_arp
                         value: "true"
                       - name: address
-                        value: "${CONTROL_PLANE_ENDPOINT_IP}"
+                        value: "control_plane_endpoint_ip"
                       - name: port
-                        value: "${CONTROL_PLANE_ENDPOINT_PORT=6443}"
+                        value: "control_plane_endpoint_port"
```

Comment:
nit: pls file a tracker to add them as vars or default the values on line 63, 65 and 67 below

Reply:
Also as discussed in future we may have to make the entire section as variable if user want to update any other variables which we are not sed-ing yet

Reply:
as discussed earlier, we will add later in separate PR if needed